templates/openwisp2/urls.py

```diff
@@ -17,15 +17,18 @@
     {% endif %}
```

Comment:
since you're reindenting these lines, check the `if openwisp2_controller_urls` which was added recently